src/test/util_tests.cpp

```diff
@@ -2508,19 +2508,19 @@ BOOST_AUTO_TEST_CASE(test_tracked_vector)
     BOOST_CHECK(v1[0].origin == &t1);
     BOOST_CHECK_EQUAL(v1[0].copies, 1);
 
-    auto v2 = Vector(std::move(t2));
+    auto v2 = Vector(std::move(*&t2));
     BOOST_CHECK_EQUAL(v2.size(), 1U);
     BOOST_CHECK(v2[0].origin == &t2);
```

Comment:
Shouldn't you just write `// NOLINT(*-use-after-move)` to disable the check for these lines (which also documents that you're deliberately doing weird things) ? https://clang.llvm.org/extra/clang-tidy/#suppressing-undesired-diagnostics

Reply:
Makes sense, I agree `// NOLINT(*-use-after-move)` looks better than what I suggested (`SafelyMoveFrom(t2)` or `std::move(*t2)`). I didn't know what syntax to disable linter was, but I assumed it would be worse than that.

I think ideally the compiler or linter would have a usable-after-moved directive that could be applied to types like `std::vector` where using after move is safe and well defined, so individual moves wouldn't have to be annotated. But either way works

Reply:
Switched to `// NOLINT`.